ext/storage_sources/s3_store/s3_storage_source.cpp

```diff
@@ -87,7 +87,6 @@ struct S3_FILE_HANDLE {
 };
 
 /* Configuration variables for connecting to S3CrtClient. */
-const Aws::String region = Aws::Region::AP_SOUTHEAST_2;
 const double throughputTargetGbps = 5;
 const uint64_t partSize = 8 * 1024 * 1024; /* 8 MB. */
```

Comment:
Would it be useful to also put these into the config string?

Reply:
Probably, but we are short of time now, so we are putting only the ones that are really required. Once this phase of the project is over, we will likely re-evaluate this question.